client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-expanded-content/test/boost-site-performance.test.tsx

```diff
@@ -59,6 +59,40 @@ describe( 'BoostSitePerformance', () => {
 		expect( strongTag ).toHaveStyle( 'font-weight: bold' );
 	} );
 
+	test( 'renders the Optimize css button when there is a score and has boost', () => {
+		const hasBoost = true;
+		render(
+			<BoostSitePerformance
+				boostData={ boostData }
+				hasBoost={ hasBoost }
+				siteId={ siteId }
+				siteUrlWithScheme={ siteUrlWithScheme }
+				trackEvent={ trackEventMock }
+				hasError={ false }
+			/>
+		);
+
+		const button = screen.getByRole( 'link', { name: /optimize css/i } );
+		expect( button ).toBeInTheDocument();
+	} );
+
+	test( 'renders the Configure Boost button when there is a score and has no boost', () => {
+		const hasBoost = false;
+		render(
+			<BoostSitePerformance
+				boostData={ boostData }
+				hasBoost={ hasBoost }
+				siteId={ siteId }
+				siteUrlWithScheme={ siteUrlWithScheme }
+				trackEvent={ trackEventMock }
+				hasError={ false }
+			/>
+		);
+
+		const button = screen.getByRole( 'link', { name: /configure boost/i } );
+		expect( button ).toBeInTheDocument();
```

Comment:
I think we could test this further:

- Check if the button has the correct URL.
- Simulate a click and see if `trackEventMock` is called with the correct event name.

Reply:
This has been updated 👍